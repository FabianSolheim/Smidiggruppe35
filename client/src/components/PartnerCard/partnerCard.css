.PartnerCardBox {
    background-color: rgba(3,79,122,0.1);
}
