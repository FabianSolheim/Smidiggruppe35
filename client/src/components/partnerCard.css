.partnerCard {
    text-align: center;
}
.centerDiv {
    margin: auto;
    width: 90%;
    background-color: #081825;
    padding: 10px;
    color: white;
}
.items {
    text-align: center;
}