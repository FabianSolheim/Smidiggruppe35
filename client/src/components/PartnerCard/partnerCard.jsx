import React from "react";
import "./partnerCard.css";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";

export function PartnerCard() {
  return (
    <Box className="PartnerCardBox">
      <Grid container spacing={2} style={{ maxWidth: 1440, margin: "auto" }}>
        <Grid item xs={12} md={4}>
          <div style={{ textAlign: "center", marginTop: 10 }}>
            <img src="https://i.postimg.cc/j5yt5mSV/Kristiania.png" />
          </div>
        </Grid>
        <Grid item xs={12} md={4}>
          <div style={{ textAlign: "center", margin: 20 }}>
            <img src="https://i.postimg.cc/bwYJ5yNZ/linkedin.png" />
          </div>
        </Grid>
        <Grid item xs={12} md={4}>
          <div style={{ textAlign: "center", margin: 20 }}>
            <img src="https://i.postimg.cc/c404Kx3F/google.png" />
          </div>
        </Grid>
      </Grid>
    </Box>
  );
}
