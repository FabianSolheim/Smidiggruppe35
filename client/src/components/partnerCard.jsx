import React from "react"
import "./partnerCard.css"
import Box from '@mui/material/Box';
import Grid from '@mui/material/Grid';

export function PartnerCard(){
    return (
        <div className="centerDiv">
        <Box sx={{ flexGrow: 1 }}>
            <Grid container spacing={2}>
                <Grid item xs={4} md={4}>
                    <div className="items">Google</div>
                </Grid>
                <Grid item xs={4} md={4}>
                    <div className="items">HÃ¸yskolen Kristiania</div>
                </Grid>
                <Grid item xs={4} md={4}>
                    <div className="items">Linkedin</div>
                </Grid>
            </Grid>
        </Box>
        </div>
    );
}