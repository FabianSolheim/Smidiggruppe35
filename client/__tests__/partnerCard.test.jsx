import React from "react";
import ReactDOM from "react-dom";
import { PartnerCard } from "../src/components/PartnerCard/partnerCard";

describe("PartnerCard test", () => {
  it("shows partnerCards", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<PartnerCard />, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
