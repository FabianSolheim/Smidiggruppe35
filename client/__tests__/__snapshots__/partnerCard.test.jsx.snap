// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PartnerCard test shows partnerCards 1`] = `"<div class=\\"PartnerCardBox MuiBox-root css-0\\"><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root\\" style=\\"max-width: 1440px;\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin-top: 10px;\\"><img src=\\"https://i.postimg.cc/j5yt5mSV/Kristiania.png\\"></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin-top: 20px;\\"><img src=\\"https://i.postimg.cc/bwYJ5yNZ/linkedin.png\\"></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin-top: 20px;\\"><img src=\\"https://i.postimg.cc/c404Kx3F/google.png\\"></div></div></div></div>"`;
